src/clips-specs/rcll-central/goal-reasoner.clp

```diff
@@ -828,6 +841,19 @@
   =>
   (printout (log-debug ?v) "Evaluate " ?class " goal " ?goal-id crlf)
   (set-robot-to-waiting ?robot)
+  ;if a move-out-of-the-way has urgent high priority, reset it on goal completion
+  (do-for-fact ((?f wm-fact))
+	(wm-key-prefix ?f:key (create$ monitoring move-out-of-way high-prio))
+	(do-for-all-facts ((?og goal) (?ogm goal-meta))
```

Comment:
do we actually need this?
The goal is not assigned to a robot anymore after this rule, this should trigger the priority reset already (rule goal-production-change-priority-move-out-of-way)
Hence i would remove the innder do-for-all-facts